doc/source/analyzing/fields.rst

```diff
@@ -455,6 +455,8 @@ General Particle Fields
 Every particle will contain both a ``particle_position`` and ``particle_velocity``
 that tracks the position and velocity (respectively) in code units.
 
+.. FIXME: Update the following sections to reflect differences in yt-4.0.
```

Comment:
Rather than leaving this comment here can you file an issue? That will be less likely to be missed.

Reply:
Sure, will do.

Reply:
It's tracked in #2218 now.